scripts/configure_libmesh.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+#* This file is part of the MOOSE framework
+#* https://www.mooseframework.org
+#*
+#* All rights reserved, see COPYRIGHT for full restrictions
+#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+#*
+#* Licensed under LGPL 2.1, please see LICENSE for details
+#* https://www.gnu.org/licenses/lgpl-2.1.html
+
+# Configure libMesh with the default MOOSE configuration options
+#
+# Separated so that it can be used across all libMesh build scripts:
+# - scripts/update_and_rebuild_libmesh.sh
+# - conda/libmesh/build.sh
+function configure_libmesh()
+{
+  if [ -z "$SRC_DIR" ]; then
+    echo "SRC_DIR is not set for configure_libmesh"
+    exit 1
+  fi
+  if [ ! -d "$SRC_DIR" ]; then
+    echo "$SRC_DIR=SRC_DIR does not exist"
+    exit 1
+  fi
+
+  if [ -z "$METHODS" ]; then
+    echo "METHODS must be set in configure_libmesh"
+    exit 1
+  fi
+
+  if [[ -n "$INSTALL_BINARY" ]]; then
+    echo "INFO: INSTALL_BINARY set"
+  else
+    export INSTALL_BINARY="${SCRIPT_DIR}/../libmesh/build-aux/install-sh -C"
+  fi
```

Comment:
@loganharbour this bit could remain in `update_and_rebuild_libmesh.sh`, but we then would need to make sure that `INSTALL_BINARY` is set in `build.sh` as well. If I leave this here, it might be worth adding a comment in the old location so that this can be easily found again by someone looking at the update script. Thoughts?

Reply:
Comment added in 7763178